### DISCUSSION TITLE
I have one question. why?

### DISCUSSION THREAD

**Discussion by Shibva** [HbmMods/Hbm-s-Nuclear-Tech-GIT#659] [Q&A]
I finally found a mod that would have been capable of doing the job other than OpenComputers now that you support to it, but unfortunately, I cant seem to get it to be able to connect to the machine  remove the damn template from the slot, neither could I with open computers using the transporter

Is this on purpose?

**Comment by SoB1nary** [comment]
your question is too clear. please try being more cryptic

**Comment by Shibva** [comment]
I think I answered it myself

he did do it most likey on purpose, because he knew that people would try to find a way to automate the assembler and the chemplant in a fashion that would make the person only need one to automate

I may be wrong but that would make sense

**Reply by SoB1nary** [reply]
what
